### DISCUSSION TITLE
Best practice on handling MF packages in SavyCAN?

### DISCUSSION THREAD

**Discussion by spot2000** [collin80/SavvyCAN#638] [General]
Hi, most of the packages i try and decode is MF, meaning that i get all  my answers from CAN bus in MF, ie more than one package as ansvwer.

What is best way to find the usefull bytes or bits in that flow, up until now i export it to excel and then assembly the multi frame response to single line answers, can that be done in SavyCAN or do anyone else do the same, and what tools do you use for it?

**Comment by spot2000** [comment]
Would it be easy to implement a feature that combines MF answers to a single row answer and also make it possible to use all the other analysis tools that are in SavyCAN also with MF answers?

I need to look for hard to find bit or byte responsens in large MF flows and right now I cant make any good use of SavyCAN for that analyse and need to do them in excel and that is too time consuming.

**Comment by spot2000** [comment]
Today MF answers are combined togheter in one row when doing UDS scans, but If I for a long time want to analyse massive MF answer for one specifik CAN ID, UDS SID, UDS DID it would be better to do it in SavyCAN instead of excel.

**Comment by gsegov** [comment]
Hello! I don't know if my question is for this topic but I will give a try.
Is it possible Savycan to read binary files and translate it to human read format (example: If I would like to understand in wich ID is the DPF sensors)?

**Comment by rusoku2** [comment]
How will the bin file be created?

**Reply by gsegov** [reply]
I downloaded with PCMFlash software from automotive Engine Controll Unit (ECU)
